### PR TITLE
adding static maps to cartodb.js docs

### PR DESCRIPTION
**Pull request by ohasselblad**
Waiting to add an example of the `basemap` option, but rest is ready for review!

Ref: #371 

cc @javierarce @javisantana

### CODE REVIEW

File: doc/API.md [hunk 1]
```diff
@@ -960,6 +960,192 @@ sql.getBounds('select * from table').done(function(bounds) {
...
+
+### Quick Start
+
+The easiest way to generate an image is by using the following piece of code, which generates is replaced by an `img` tag once run in an HTML file:
+
+```javascript
+<script>
+var vizjson_url = 'https://documentation.cartodb.com/api/v2/viz/008b3ec6-02c3-11e4-b687-0edbca4b5057/viz.json';
+
+cartodb.Image(vizjson_url, {override_bbox: true})
```
Comment (javierarce) [inline doc/API.md]: The `{override_bbox: true}` part shouldn't be necessary.

File: doc/API.md [hunk 2]
```diff
@@ -960,6 +960,192 @@ sql.getBounds('select * from table').done(function(bounds) {
...
+
+- **boundingBox**: an array of coordinates making up the bounding box for your map. `boundingBox` takes the form: `[sw_lat, sw_lon, ne_lat, ne_lon]`.
+
+##### Returns
+
+An _Image_ object
+
+#### Image.into(HTMLImageElement)
+
+Inserts the image into the HTML DOM element specified.
```
Comment (javierarce) [inline doc/API.md]: I think we could add a small example here.

For example something like this:
https://github.com/CartoDB/cartodb.js/wiki/CartoDB-Map-API#imageintohtmlimageelement

Comment (ohasselblad) [inline doc/API.md]: :+1:

File: doc/API.md (hunk 1 above)
Comment (ohasselblad) [inline doc/API.md]: :+1:

**Comment by ohasselblad** [comment]
@javierarce I think I got all of the edits requested! Let me know if there are others!

**Comment by javierarce** [comment]
I think it's perfect now. It's good to go! :)

**Comment by ohasselblad** [comment]
:bike:

**Comment by rochoa** [comment]
:muscle:

**Pull request merged**